src/Widgets/ImageWidget/ImageWidgetComponent.js

```diff
@@ -1,26 +1,42 @@
 import * as React from "react";
 import * as Scrivito from "scrivito";
+import AOS from "aos";
 
-Scrivito.provideComponent("ImageWidget", ({ widget }) => {
-  let image = (
-    <Scrivito.ImageTag
-      content={widget}
-      attribute="image"
-      alt={alternativeText(widget)}
-    />
-  );
-
-  const link = widget.get("link");
-  if (link && !Scrivito.isInPlaceEditingActive()) {
-    image = <Scrivito.LinkTag to={link}>{image}</Scrivito.LinkTag>;
+class ImageWidget extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    AOS.init();
```

Comment:
Please move `AOS.init();` into [`componentDidMount`](https://reactjs.org/docs/react-component.html#componentDidMount) since it should be called _after_ the first things are rendered.